Review vignette

Robinlovelace opened the pull request.

Robinlovelace commented on vignettes/osmdata.Rmd:
````diff
@@ -49,11 +49,11 @@ An `osmdata` query consists of an initial `opq()` query and one or more features
 osmdata_sp (q)
 ```
-or an `R Sinple Features` objects with
+or an `R Simple Features` objects with
````
@mpadge I think it would be better to show the user first how to download the raw data onto the hd before converting it - more bandwidth efficient and a chance to show the `request -> dl -> conversion` steps in process. Happy to do this but cannot remember the function to simple dl raw OSM data.

Robinlovelace replied:
That way the data is useful regardless of if you use R for analysis or not, as shown here:

![osm-qgis](https://cloud.githubusercontent.com/assets/1825120/23326333/7189dd04-faf4-11e6-91f1-bd1b0c366cad.png)

mpadge commented:
Thanks Robin! Could you please just make one modification: `osmdata _xml()` has a file argument to enable direct downloading without needing your `writeLines` example. Files can also be passed directly to all `osmdata...()` functions.

Robinlovelace commented:
Thanks @mpadge - I've fixed that issue and removed the duplication (sorry, should have read ahead!).

Still more to go. I think making `dat_sp` and `dat_sf` separate is worthwhile. Do you want to iterate on this version (check out the QGIS image) or shall I carry on? I slightly prefer the former option as it takes the pressure off me + allows quick iteration.
